pkg/orchestration/wiring/postgres/plugin.go

```diff
@@ -72,22 +72,32 @@ func New() *WiringPlugin {
 	}
 }
 
+// getRDSDependency returns a single RDS dependency if found in a given list, returning an error if multiple are found
 func getRDSDependency(dependencies []wiringplugin.WiredDependency) (wiringplugin.WiredDependency, error) {
```

Comment:
Use https://github.com/atlassian/voyager/blob/13530e71e9fb03823878012ae88316d52c52c9d4/pkg/orchestration/wiring/wiringplugin/types.go#L33 to get the only dependency and then look for the needed shape in it.